packages/app-bridge/src/types/Command.ts

```diff
@@ -0,0 +1,26 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Asset } from './Asset';
+
+export enum Command {
+    AssetViewer = 'AssetViewer',
+    AssetChooser = 'assetChooser',
+}
```

Comment:
it's probably better to avoid enums in these case, since it's dangerous to export this through types as it's not only a type (compile team) but also javascript code (runtime).

Reply:
Removed this and used direct values